packages/core/src/routes/session.ts

```diff
@@ -67,6 +67,9 @@ export default function sessionRoutes<T extends AnonymousRouter>(router: T, prov
       }),
     }),
     async (ctx, next) => {
+      const { jti } = await provider.interactionDetails(ctx.req, ctx.res);
+      ctx.log.sessionId = jti;
```

Comment:
since those are common session check logics, should we extract as a middleware and applied to specific routes? Like kogAuth cc: @wangsijie @gao-sun

Reply:
Agree

Reply:
LOG-2256 Do it later